test/acceptance/application-test.js

```diff
@@ -43,15 +43,16 @@ describe('Acceptance | application', function() {
       promoteHtmlbars();
 
       return app.create('dummy', {
-        fixturesPath: 'tests'
+        fixturesPath: 'tests',
+        yarn: true
```

Comment:
perhaps we should leave this test intact and make a new one that tests yarn. Then both our npm and yarn code paths get tested.